data/tokens.json

```diff
@@ -2887,10 +2895,18 @@
         "value": "{gtmds_color_white}",
         "type": "color"
       },
-      "gtmds_toast-close-icon_color": {
+      "gtmds_toast-dismiss-button_color": {
         "value": "rgba({Toast.gtmds_toast_color}, 0.5)",
```

Comment:
I saw this color is used with opacity, while we were discussing initially to avoid opacity. What is the reason for that?

Reply:
reason is that we have very colorful situation and want to introduce one thing that would work everywhere but we need opacity as we don't want a color that is as strong as the rest of the content